### PR TITLE
Set primary skin color quit popin

### PR DESCRIPTION
**Pull request by antonia2929**
Trello card : [[OR] Popin "Quit session" CTA Primary color](https://trello.com/c/fvlw8l35/2815-or-popin-quit-session-cta-primary-color)

**Detailed purpose of the PR**
Set the color of primary quit popin button "continue learning" to the primary skin color of the platform.

**Result and observation**

### After
Capture d’écran 2022-11-16 à 11 46 55

**Testing Strategy**
- [x] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-app-review/src/types/common.ts [hunk 1]
```diff
@@ -218,3 +219,9 @@ export type WithRequired<T, K extends keyof T> = T & {
   // the "-" is a Mapping Modifier, removes optionality from a prop
   [P in K]-?: T[P];
 };
+
+export type PrimarySkin = {
```
Comment (jomaora) [inline packages/@coorpacademy-app-review/src/types/common.ts:223]: Skin

File: packages/@coorpacademy-app-review/src/views/slides/test/button-revising.on-click.test.ts [hunk 2]
```diff
@@ -24,7 +24,7 @@ import {qcmGraphicSlide} from './fixtures/qcm-graphic';
 import {sliderSlide} from './fixtures/slider';
 import {templateSlide} from './fixtures/template';
 
-const connectedOptions = {translate, onQuitClick: identity};
+const connectedOptions = {translate, onQuitClick: identity, skin: {common: {primary: '#248e59'}}};
```
Comment (jomaora) [inline packages/@coorpacademy-app-review/src/views/slides/test/button-revising.on-click.test.ts:27]: fais une fixtrure de la Skin et tu l'importes dans les test où t'as besoin

**Review by jomaora — CHANGES_REQUESTED** [review]

**Review by StefanoA1 — APPROVED** [review]

File: packages/@coorpacademy-app-review/src/views/slides/test/fixtures/skin.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import {Skin} from '../../../../types/common';
+
+export const skin: Skin = {
```
Comment (scalabw) [inline packages/@coorpacademy-app-review/src/views/slides/test/fixtures/skin.ts:3]: use the same as the one in Sandbox ? or make a generic one ?

File: packages/@coorpacademy-app-review/src/views/slides/test/header.on-click.test.ts [hunk 4]
```diff
@@ -43,12 +43,17 @@ const state: StoreState = {
 test('should dispatch OPEN_POPIN action after a click on close button in header', async t => {
   const expectedAction = [{type: OPEN_POPIN}];
   const {dispatch, getState} = createTestStore(t, state, {services}, expectedAction);
-  const props = mapStateToSlidesProps(getState(), dispatch, {translate, onQuitClick: identity});
+  const props = mapStateToSlidesProps(getState(), dispatch, {
+    translate,
+    onQuitClick: identity,
+    skin: {common: {primary: '#248e59'}}
```
Comment (scalabw) [inline packages/@coorpacademy-app-review/src/views/slides/test/header.on-click.test.ts:49]: same here ?

File: packages/@coorpacademy-app-review/src/views/slides/test/on-quit-popin.on-click.test.ts [hunk 5]
```diff
@@ -69,7 +70,8 @@ test('should dispatch onQuitClick function via the property handleOnclick of fir
   const {dispatch, getState} = createTestStore(t, state, {services}, expectedAction);
   const props = mapStateToSlidesProps(getState(), dispatch, {
     translate,
-    onQuitClick: () => t.pass()
+    onQuitClick: () => t.pass(),
+    skin: {common: {primary: '#248e59'}}
```
Comment (scalabw) [inline packages/@coorpacademy-app-review/src/views/slides/test/on-quit-popin.on-click.test.ts:74]: same here

**Review by scalabw — APPROVED** [review]

**Pull request merged**